<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>551</width>
      <height>291</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>128</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Würfe</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Spieler 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Spieler 2</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>Nur EINSER</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>Nur ZWEIER</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>Nur DREIER</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>Nur VIERER</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>Nur FUENFER</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>Nur SECHSER</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>GESAMT -&gt;</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <brush brushstyle="Dense4Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="Dense4Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="Dense4Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>Bonus bei 63+</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>Plus 35 Pkt.</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>Plus 35 Pkt.</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>GESAMT + BONUS</string>
     </property>
     <property name="background">
      <brush brushstyle="Dense5Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="Dense5Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="Dense5Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>Dreierpasch</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>Viererpasch</string>
     </property>
    </item>
    <item row="10" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>Fullhouse (25)</string>
     </property>
    </item>
    <item row="11" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="11" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>kleine Straße (30)</string>
     </property>
    </item>
    <item row="12" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="12" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>große Straße (40)</string>
     </property>
    </item>
    <item row="13" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="13" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="14" column="0">
     <property name="text">
      <string>KNIFFEL (50)</string>
     </property>
    </item>
    <item row="14" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="14" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="15" column="0">
     <property name="text">
      <string>Chance</string>
     </property>
    </item>
    <item row="15" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="15" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="16" column="0">
     <property name="text">
      <string>ges. unt. Teil</string>
     </property>
     <property name="background">
      <brush brushstyle="Dense4Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="16" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="Dense4Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="16" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="Dense4Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="17" column="0">
     <property name="text">
      <string>ges. ob. Teil</string>
     </property>
     <property name="background">
      <brush brushstyle="Dense4Pattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="17" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="Dense4Pattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="17" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="Dense4Pattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="18" column="0">
     <property name="text">
      <string>GESAMT -&gt;</string>
     </property>
     <property name="background">
      <brush brushstyle="Dense5Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="18" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="Dense5Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="18" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="Dense5Pattern">
       <color alpha="255">
        <red>125</red>
        <green>125</green>
        <blue>125</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="roll_btn">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>465</y>
      <width>106</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Würfeln!!!!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="first_roll">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>50</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#first_roll{
background-color: transparent;
border-image: url(:1.png);
background: none;
background-repeat: none;
}
#first_roll:pressed
{
	border-image: url(:1_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="second_roll">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>51</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="third_roll">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>50</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fourth_roll">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>50</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fifth_roll">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>50</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="np_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>465</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Nächster Spieler</string>
    </property>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-30</y>
      <width>451</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt; font-weight:600;&quot;&gt;KNIFFELLLLL!!!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
